### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by AdamJahur**
added a button to link to google calendar so event can be added to personal calendar and shared with others.  a lot of the changes i made was for indentation so thats why theres so many.

### CODE REVIEW

**Comment by AdamJahur** [comment]
i havent styled the button yet i jus wanted to make sure it works

**Comment by Nickpopadopolous** [comment]
If anything we can just go back a branch

**Comment by AdamJahur** [comment]
i dont think we will need to

**Comment by Nickpopadopolous** [comment]
Sounds good!

File: secondPage.html [hunk 1]
```diff
@@ -88,140 +55,192 @@ <h3 id="movieTitle">No Title</h3>
...
 							<label for="middle-label" class="date-picker">Zip Code: </label>
+
 						</div>
-	
+
 						<div class="small-9 columns">
 
 							<form action="javascript:0">
 
-								<input type="text" required pattern="(\d{5}([\-]\d{4})?)" value name="postalcodes_pattern0" id="middle-label" list="postalcodes_pattern0_datalist" placeholder="12345">
```
Comment (LibbyKnight) [inline secondPage.html]: why was this removed?

Comment (AdamJahur) [inline secondPage.html]: its right under it it was showing up twice in my code

Comment (AdamJahur) [inline secondPage.html]: look at 111

Comment (AdamJahur) [inline secondPage.html]: it was showing up twice if u look under it, its still there

Comment (AdamJahur) [inline secondPage.html]: okay ill fix it back

**Comment by AdamJahur** [comment]
okay fixed it

**Comment by LibbyKnight** [comment]
looks awesome! =)

**Comment by Nickpopadopolous** [comment]
Looks good!
